packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.jsx

```diff
@@ -285,26 +282,14 @@ const ProofOfAddressForm = ({
                     {form_state.should_show_form && (
                         <form noValidate className='account-form' onSubmit={handleSubmit}>
                             <FormBody scroll_offset={isMobile() ? mobile_scroll_offset : '80px'}>
-                                <FormSubHeader
-                                    title={is_dashboard ? localize('1. Address') : localize('Details')}
-                                    subtitle={is_dashboard && '(All fields are required)'}
-                                />
+                                <FormSubHeader title={localize('1. Address')} subtitle={'(All fields are required)'} />
                                 <FormBodySection
-                                    has_side_note={is_dashboard}
+                                    has_side_note={true}
```

Comment:
You can write this simpler:

```suggestion
                                    has_side_note
```

Reply:
Aah I see! Fixed it.